unasync.py

```diff
@@ -15,13 +15,15 @@
     ('aclose', 'close'),
     ('aclose_func', 'close_func'),
     ('aiterator', 'iterator'),
+    ('lookup_async_backend', 'lookup_sync_backend'),
     ('__aenter__', '__enter__'),
     ('__aexit__', '__exit__'),
     ('__aiter__', '__iter__'),
     ('@pytest.mark.anyio', ''),
+    (r'@pytest.fixture\(params=\["auto", "anyio"\]\)',
+     '@pytest.fixture(params=["sync"])'),
     ('lookup_async_backend', "lookup_sync_backend"),
-    ('auto', 'sync'),
-    ('anyio', 'sync'),
+    ('auto', 'sync')
```

Comment:
(Nit) To minimize the diff (also keeping trailing commas is git-friendly)

```suggestion
    ('auto', 'sync'),
```

Actually, do we still need this replacement?